components/portal/widgets/controllers.js

```diff
@@ -94,6 +96,10 @@ define(['angular', 'moment'], function(angular, moment) {
             $scope.widget = data;
             $scope.widgetType = widgetType(data);
           }
+          // If there's a remove button, make it focusable by keyboard
+          if ($transclude.isSlotFilled('removeButton')) {
```

Comment:
What if there's not a remove button, but there is other content in that menu? An about link? A widget-specific help link? Like, I'm not sure what all this menu might be used for in the future. Is this solution going to work if there's something interesting in this menu but that something isn't a remove button?

Reply:
>Is this solution going to work if there's something interesting in this menu but that something isn't a remove button?

No, it won't work if there's something else. We could use an unnamed `transclude` as well to allow any random content, but since the contextual menu introduces a change in how that content is presented, I'd suggest avoiding that until there's a use case for it, since it would be difficult/impossible to ensure that such content would have a good UX for screen readers and keyboard users.